doc/changes/latest.inc

```diff
@@ -31,6 +31,8 @@ Enhancements
 
 - Add sEEG source visualization using :func:`mne.stc_near_sensors` and sEEG working tutorial by `Eric Larson`_ and `Adam Li`_ (:gh:`8402`)
 
+- If a ``baseline`` tuple containing one or two ``None`` values – e.g. ``(None, 0)``, ``(0, None)``, or ``(None, None)`` – is passed to `~mne.Epochs` or `~mne.Epochs.apply_baseline`, the ``None`` value(s) will be replaced with the actual time (i.e., :attr:`~mne.Epochs.tmin` and :attr:`~mne.Epochs.tmax`, respectively) when populating :attr:`~mne.Epochs.baseline`, by `Richard Höchenberger`_ (:gh:`8442`)
```

Comment:
just chiming in that this would be great to have!

Reply:
will continue working on this soon.